apps/hash-api/src/graph/knowledge/system-types/user.ts

```diff
@@ -435,6 +439,27 @@ export const createUser: ImpureGraphFunction<
     ],
   );
 
+  const hashInstanceAdminsAccountGroupId =
+    await getHashInstanceAdminAccountGroupId(ctx, authentication);
+
+  /** Grant permissions to hash instance admins to edit the user entity */
+  await modifyEntityAuthorizationRelationships(ctx, authentication, [
```

Comment:
Good point, have addressed in https://github.com/hashintel/hash/pull/4203/commits/c659cc62d2c4de80ac2c1bb8a67661f174201cf6.

> Something that is already attempting to do this is further above in the code, but I suspect it has broken since the HASH Instance entity is no longer owned by the HASH instance account group (it's owned by the HASH org) – should be fixable by getting the accountGroupId how you are here.

I fixed this in the same commit. I assume this means there might be some user entities in production where the HASH instance admin group doesn't have the `administrator` permission, if so is this something you'd like me to address in the migration? Or ticketize?

Reply:
We don't need both `editor` and `administrator`, because `administrator` implies `editor` – I'd just give `administrator`

Reply:
Ah nice, then there isn't an issue. Have addressed in https://github.com/hashintel/hash/pull/4203/commits/72509f53af92c8ace1b1cb063ca0f0541b024e0a and tested it out locally 👍